datalad/resources/website/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Datalad Repository</title>
+        <link rel="shortcut icon" type="image/x-icon" href=".git/datalad/web/assets/images/favicon-16x16.png" />
+        <link rel="stylesheet" type="text/css" href=".git/datalad/web/assets/css/jquery.dataTables.css">
+        <style type="text/css" class="init">
+          body{
+           max-width: 1200px;
+           margin: 2em auto;
+         }
+         td.dir {font-weight: bold; color: #00A1CB; }
+         td.git {font-weight: bold; color: #00BA88; }
+         td.annex {font-weight: bold; color: #EC6C20; }
+         td.link-broken {color:#E53B51; }
+         td.link {color: #FFD03C;  }
+         td.file {color: #67CDDC;  }
+        </style>
+    </head>
+    <body>
+        <h1><img src='.git/datalad/web/assets/images/datalad_banner.svg' width="196"></h1>
```

Comment:
The `<img>` needs the `alt` attribute, even more-so since you're playing the edge of semantics/accessibility by having no content in the `<h1>`.